helmholtz-analytics/heat
# Update latest release changes to main

## Due Diligence

- General:
    - [ ]  **base branch** must be `main` for new features, latest release branch (e.g. `release/1.3.x`) for bug fixes
    - [ ]  **title** of the PR is suitable to appear in the [Release Notes](https://github.com/helmholtz-analytics/heat/releases/latest)
- Implementation:
    - [ ] unit tests: all split configurations tested
    - [ ] unit tests: multiple dtypes tested

## Description





Issue/s resolved: #

## Changes proposed:
* Support latest PyTorch release
* add checklist to PR description
* add pytorch 2.0 to list
* Support latest PyTorch release (#1311)
* activate carry forward flag
* start pr test only on path heat/**
* update test environment settings
* [pre-commit.ci] auto fixes from pre-commit.com hooks
for more information, see https://pre-commit.ci
* skip sparse test mul
* read envar for skipping

## Type of change


## Memory requirements


## Performance


#### Does this change modify the behaviour of other functions? If so, which?
yes / no
